doc/api/magic-plot.md

```diff
@@ -160,3 +160,100 @@ ax = %sqlplot histogram --table no_nulls --column body_mass_g --with no_nulls
 ax.set_title("Body mass (grams)")
 _ = ax.grid()
 ```
+## `%sqlplot bar`
+
+Shortcut: `%sqlplot bar`
+
+`-t`/`--table` Table to use (if using DuckDB: path to the file to query)
+
+`-c`/`--column` Column to plot.
+
+`-o`/`--orient` Barplot orientation (`h` for horizontal, `v` for vertical)
+
+`-w`/`--with` Use a previously saved query as input data
+
+`-sn`/`--show_num` Show number on top of the bar
```

Comment:
to be consistent with the other options, let's only use one-character options, this can be `-S` (capital S)

Reply:
also, long options should not contain underscores of abbreviations. so this should be `--show-numbers`